src/components/BlogList.vue

```diff
@@ -65,24 +31,23 @@ onMounted(() => {
   fetchArticles();
 });
 
-watch(
-  () => props.searchQuery,
-  (newQuery) => {
-    page.value = 1;
-    fetchArticles();
-  }
-);
+function formatDate(timestamp: number) {
+  const date = new Date(timestamp * 1000);
+  return new Intl.DateTimeFormat('en-US', {
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric',
+  }).format(date);
+}
 </script>
 
-
-
 <template>
   <div class="blog-list">
     <div 
-    v-for="(article, index) in articles" 
-    :key="index" 
-    class="blog-item"
-    @click="navigateToBlog(article.id)" 
+      v-for="(article, index) in articles" 
+      :key="index" 
+      class="blog-item"
+      @click="navigateToBlog(article.id)" 
```

Comment:
```suggestion
      @click="onArticleSelect(article.id)" 
```

Following html element event names (onclick, onblur, onfocus) we should follow the same convention for our custom function names.